work/Maya/exersize2_2.py

```diff
@@ -2,21 +2,30 @@
 
 import os
 import signal
+import sys
 
 def start_process():
-    pid = os.fork()
-    if pid < 0:
-        print("Fork failed")
-        os.exit(-1)
-    elif pid > 0:
-        print(f"Parent process: {os.getpid()}")
-        return pid
-    else:
-        os._exit(0)
+    try:
+        pid = os.fork()
+        if pid < 0:
+            print("Fork failed")
+            os.exit(-1)
+        elif pid > 0:
+            print(f"Parent process: {os.getpid()}")
+            return pid
+        else:
+            os._exit(0)
+    except OSError as e:
+        print(f"Fork failed: {e}")
+        os._exit(1)
 
 def simulate_kill_9(pid):
-    os.kill(pid, signal.SIGKILL)
-    print(f"Process with PID {pid} killed")
```

Comment:
the script dont have to do fork and nothing else, only use os.kill(pid)

Reply:
to use it you can start a daemon o a inifinitie process ( one with a `while True: pass`)